.ci/pipelines/build.Jenkinsfile

```diff
@@ -34,19 +34,24 @@ pipeline {
                         sh 'make -C .ci TARGET=ci ci'
                     }
                 }
+                stage('Run ci checks prior to build') {
+                    steps {
+                        sh 'make -C .ci TARGET=ci-check ci'
```

Comment:
I think this step is redundant we already have it on line 29 https://github.com/elastic/cloud-on-k8s/pull/6143/files#diff-68c3b03af5147c61c49f4b3316dcdd00fc6c3566986ddf0a12f30f3346cb95a5L29

Reply:
Good point @pebrc ; removed.  Interesting, as it's included in `ci-release` target, so we were running it twice, it seems.